projects/plugins/jetpack/_inc/client/sharing/share-buttons.jsx

```diff
@@ -54,7 +55,7 @@ export const ShareButtons = withModuleSettingsFormHelpers(
 
 				if ( shouldShowSharingBlock ) {
 					cardProps.href = `${ siteAdminUrl }site-editor.php?path=%2Fwp_template`;
-				} else if ( isLinked && ! isOfflineMode ) {
+				} else if ( isLinked && ! isOfflineMode && ! siteUsesWpAdminInterface ) {
```

Comment:
[nit] We could also add "- Is the site using the classic admin interface?" to the comment above.

Reply:
Good call!